Add HW1

eogenio777 opened the pull request.

eogenio777 committed:
Add HW1

akulebyakin commented on out/test/selenium_homeworks/addSubTestNG.xml:
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```
Please add `out` directory to `.gitignore` and remove it from github repository

eogenio777 replied:
The `.gitignore` file is changed, the `out` directory is removed.

akulebyakin commented on src/test/java/com/epam/tc/hw1/CalculatorAddTests.java:
```diff
@@ -0,0 +1,21 @@
+        groups = {"Add Test"}
+        )
+    public void defaultSumTest(long a, long b, long expected) {
+        Calculator calculator = new Calculator();
```
Lets move Calculator initialization to the base class since you use this `Calculator calculator = new Calculator();` in each test.

eogenio777 replied:
The base `CalculatorTests` class is created with `calculator` initialization in its constructor. All test classes are inherited from it.

akulebyakin reviewed (changes requested):
Please review comments

eogenio777 committed:
Change calculator initialization

Init of the calculator is now in the base class CalculatorTests, "out" folder is added to gitignore.

eogenio777 committed:
remove unnecessary calculator import

akulebyakin reviewed (approved):
Approved. Good job!
